E, [2024-09-04T18:03:31.035522 #256458] ERROR -- : Error occurred: Field 'EU-UStID' not found
E, [2024-09-04T18:03:31.035642 #256458] ERROR -- : /var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/base.rb:45:in `block in initialize'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/base.rb:43:in `each_pair'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/base.rb:43:in `initialize'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/export.rb:15:in `new'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/export.rb:15:in `<<'
app.rb:106:in `block (2 levels) in <main>'
app.rb:105:in `each'
app.rb:105:in `block in <main>'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1804:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1804:in `block in compile!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1071:in `block (3 levels) in route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1089:in `route_eval'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1071:in `block (2 levels) in route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1120:in `block in process_route'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1118:in `catch'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1118:in `process_route'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1069:in `block in route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1066:in `each'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1066:in `route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1190:in `block in dispatch!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `catch'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `invoke'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1185:in `dispatch!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1001:in `block in call!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `catch'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `invoke'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1001:in `call!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:990:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/xss_header.rb:20:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/path_traversal.rb:18:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/json_csrf.rb:28:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/base.rb:53:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/base.rb:53:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/frame_options.rb:33:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/logger.rb:20:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/common_logger.rb:43:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:266:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:259:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/head.rb:15:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/method_override.rb:28:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/show_exceptions.rb:23:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:224:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:2115:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1674:in `block in call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1890:in `synchronize'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1674:in `call'
/var/lib/gems/3.0.0/gems/rackup-2.1.0/lib/rackup/handler/webrick.rb:111:in `service'
/var/lib/gems/3.0.0/gems/webrick-1.8.1/lib/webrick/httpserver.rb:140:in `service'
/var/lib/gems/3.0.0/gems/webrick-1.8.1/lib/webrick/httpserver.rb:96:in `run'
/var/lib/gems/3.0.0/gems/webrick-1.8.1/lib/webrick/server.rb:310:in `block in start_thread'
E, [2024-09-04T18:04:14.288268 #257021] ERROR -- : Error occurred: Value given for field 'Kennz. Korrespondenzadresse' is not a Boolean
E, [2024-09-04T18:04:14.288425 #257021] ERROR -- : /var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/field/boolean_field.rb:7:in `validate!'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/base.rb:48:in `block in initialize'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/base.rb:43:in `each_pair'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/base.rb:43:in `initialize'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/export.rb:15:in `new'
/var/lib/gems/3.0.0/gems/datev-0.11.0/lib/datev/export.rb:15:in `<<'
app.rb:106:in `block (2 levels) in <main>'
app.rb:105:in `each'
app.rb:105:in `block in <main>'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1804:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1804:in `block in compile!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1071:in `block (3 levels) in route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1089:in `route_eval'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1071:in `block (2 levels) in route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1120:in `block in process_route'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1118:in `catch'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1118:in `process_route'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1069:in `block in route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1066:in `each'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1066:in `route!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1190:in `block in dispatch!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `catch'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `invoke'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1185:in `dispatch!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1001:in `block in call!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `catch'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1161:in `invoke'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1001:in `call!'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:990:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/xss_header.rb:20:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/path_traversal.rb:18:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/json_csrf.rb:28:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/base.rb:53:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/base.rb:53:in `call'
/var/lib/gems/3.0.0/gems/rack-protection-4.0.0/lib/rack/protection/frame_options.rb:33:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/logger.rb:20:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/common_logger.rb:43:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:266:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:259:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/head.rb:15:in `call'
/var/lib/gems/3.0.0/gems/rack-3.1.7/lib/rack/method_override.rb:28:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/show_exceptions.rb:23:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:224:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:2115:in `call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1674:in `block in call'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1890:in `synchronize'
/var/lib/gems/3.0.0/gems/sinatra-4.0.0/lib/sinatra/base.rb:1674:in `call'
/var/lib/gems/3.0.0/gems/rackup-2.1.0/lib/rackup/handler/webrick.rb:111:in `service'
/var/lib/gems/3.0.0/gems/webrick-1.8.1/lib/webrick/httpserver.rb:140:in `service'
/var/lib/gems/3.0.0/gems/webrick-1.8.1/lib/webrick/httpserver.rb:96:in `run'
/var/lib/gems/3.0.0/gems/webrick-1.8.1/lib/webrick/server.rb:310:in `block in start_thread'
